corehq/apps/userreports/reports/builder/forms.py

```diff
@@ -1008,6 +1029,12 @@ def column_fieldset(self):
     @memoized
     def initial_columns(self):
         if self.existing_report:
+            reverse_agg_map = {
+                'avg': 'Average',
+                'sum': 'Sum',
+                'count': 'Count',
+                'simple': 'No Aggregation'
+            }
             cols = []
```

Comment:
@sheelio 
In data table reports expanded is the default, anything with aggregation 'simple' is expanded later on but the backend expects simple to make that happen, but the 'No Aggregation' option is the same as your count by choice. As far as the user facing language, the linked spec has changed since it was signed off and given to me but In the iteration I saw, it looked like @dimagi/product was pretty against using the phrase "Count by Choice", because they thought it would be confusing. Happy to change it, if the new language is preferred. 
Is there a reason get rid of 'Count' as an option? It seems like there is no harm in including it, but it is certainly easy to take out if need be.